maistra/bazelrc

```diff
@@ -20,6 +20,9 @@ build:aarch64 --linkopt=-fuse-ld=lld
 build:ci-config --local_ram_resources=12288
 build:ci-config --local_cpu_resources=6
 build:ci-config --jobs=3
+test:ci-config --local_ram_resources=12288
```

Comment:
This should not be necessary as `test` inherits is configuration from `build`: https://bazel.build/run/bazelrc#option-defaults